catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/FormulaTest.java

```diff
@@ -49,19 +53,23 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
 
-@RunWith(JUnit4.class)
+@RunWith(MockitoJUnitRunner.class)
 public class FormulaTest {
 
 	private List<InternToken> internTokenList;
 
 	@Before
 	public void setUp() {
-		internTokenList = new LinkedList<InternToken>();
+		internTokenList = new LinkedList<>();
+		if (ProjectManager.getInstance() == null) {
```

Comment:
PLEASE-EXPLAIN: Why is this needed?

Reply:
The project manager instance might be null - this is not checked everywhere. Might be a good moment to remove the need for it in the test. I will see what I can do :joy: